pyro/contrib/tracking/distributions.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+from torch.distributions import constraints
+
+import pyro.distributions as dist
+from pyro.distributions.torch_distribution import TorchDistribution
+from pyro.contrib.tracking.extended_kalman_filter import EKFState
+from pyro.contrib.tracking.measurements import PositionMeasurement
+
+
+class EKFDistribution(TorchDistribution):
+    r"""
+    EKF Distribution
+
```

Comment:
Could you
1. add docstrings that link to the input datatypes, and
2. ensure this class shows up on docs.pyro.ai